app/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -0,0 +1,25 @@
+Components.utils.import("resource://gre/modules/components-utils/FilterExpressions.jsm");
+Components.utils.import("resource://gre/modules/components-utils/ClientEnvironment.jsm");
+
+// Targeting
+
+async function remoteSettings(arguments) {
+
+    const TargetingContext = ChromeUtils.import(
+        "resource://messaging-system/targeting/Targeting.jsm"
+    )
```

Comment:
nit: missing semi-colon. I wonder if these are getting linted by ESLint. Possibly not if they're new .js files in a new location.

Same nit would apply to L12, L18, and L23 below.